modules/ipi-install-bmc-addressing.adoc

```diff
@@ -56,14 +56,48 @@ platform:
           disableCertificateVerification: True
 ----
 
+
+.RedFish for Dell
+
+To enable RedFish, use `redfish://` or `redfish+http://` to disable TLS. The installer requires both the hostname or the IP address and the path to the system ID. The following example demonstrates a RedFish configuration within the `install-config.yaml` file.
+
+----
```

Comment:
[source,yaml]

per our updated guidance on code blocks: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output